### ISSUE TITLE
How does this compare to Kubeflow?

### ISSUE THREAD

**Issue by suryaavala** [datarevenue-berlin/OpenMLOps#61]
I am not trying to contest this project, genuinely want to understand the reasoning/philosophy behind doing something from scratch when there are much more (relatively) mature alternatives like Kubeflow, which seem to do very similar things. 

It would be great if the maintainers of OpenMLOps could compare it with Kubeflow and other alternatives and shine some light on the philosophy of this one. 

I reckon it would be helpful for people to make more informed decisions :)

**Comment by sixhobbits** [comment]
hey @suryaavala, great question. In the initial version we actually did use Kubeflow, but we found it problematic in several ways. You can read about it here https://datarevenue.com/en-blog/kubeflow-not-ready-for-production.

TL;DR - kubeflow tries to do it all in one system, and we think doesn't get everything right quite yet (but we are following progress pretty closely). For now, we think a more flexible and modular approach is better in a lot of cases, but you are right that we should add a detailed comparison :)

**Comment by off99555** [comment]
Have you guys tried [ClearML](https://clear.ml/)? What's your opinion?

**Comment by mik3githubber** [comment]
Have you guys taken a look at Kubeflow 1.3?  That version seems to have resolved many complaints.
